pants

```diff
@@ -87,8 +93,10 @@ function exec_pants_bare() {
   # Redirect activation and native bootstrap to ensure that they don't interfere with stdout.
   activate_pants_venv 1>&2
   bootstrap_native_code 1>&2
+  # Because the venv has been activated, we simply say `python` and it will use the venv's version. We cannot use
+  # `${PY}` because it could use the wrong interpreter if the value is an absolute path.
   PYTHONPATH="${PANTS_SRCPATH_STR}:${PYTHONPATH}" RUNNING_PANTS_FROM_SOURCES=1 \
-    exec "${PY}" "${PANTS_EXE}" "$@"
+    exec python "${PANTS_EXE}" "$@"
```

Comment:
This also took a lot of time to figure out. It was broken before if you set `$PY` to an absolute path.